### PR TITLE
Add plist insecure config entry

### PR DESCRIPTION
**Pull request by deadly-panda**

### CODE REVIEW

File: PLIST_INSECURE_CONFIG/description.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+When file sharing is enabled, the entire Documents folder is used for file sharing. Files that that are not intended for user access via the file sharing feature should be stored in another part of the application's bundle. An
```
Comment (3asm) [inline PLIST_INSECURE_CONFIG/description.md:1]: The title of the KB is not correct and this description has no mention of the feature enabled.
I also don't understand the threat here.

File: PLIST_INSECURE_CONFIG/recommendation.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Make sure files containing sensitive information are not copied
```
Comment (3asm) [inline PLIST_INSECURE_CONFIG/recommendation.md:1]: This is not stating the exact config to change, it is very impractical.

File: PLIST_INSECURE_CONFIG/meta.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "risk_rating": "low",
+    "short_description": "Enabled file sharing. Files that that are not intended for user access via the file sharing feature can be accessed physically by an attacker.",
+    "references": {
+        "Common Weakness Enumeration CWE-359": "https://cwe.mitre.org/data/definitions/359.html",
+        "Apple Developer documentation":"https://developer.apple.com/documentation/bundleresources/information_property_list/uifilesharingenabled"    },
+    "title": "Enabled iTunes file sharing",
```
Comment (3asm) [inline PLIST_INSECURE_CONFIG/meta.json:7]: Title should focus on the technical attribute uiFileSharingEnabled and not iTunes.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: PLIST_INSECURE_CONFIG/description.md (hunk 1 above)
Comment (deadly-panda) [reply PLIST_INSECURE_CONFIG/description.md:1]: for the title : PLIST_INSECURE_UI_FILE_SHARING ?

**Pull request merged**